libs/components/lists/src/assets/locales/resources_en_US.json

```
@@ -1,4 +1,8 @@
 {
+  "skyux_filter_aria_label_descriptor": {
+    "_description": "Default Default aria label for a filter button when a descriptor is given",
```

```suggestion
    "_description": "Default ARIA label for a filter button when a descriptor is given",
```
Also, is "when a descriptor is given" accurate? I assume "{0}" in the `message` value is where we pull in the descriptor, but if the descriptor isn't provided, doesn't that just mean that the ARIA label will be "Filter" with no descriptor at the end? If so, I think the "when" clause is inaccurate because makes it sound like this ARIA label is only created when the descriptor is given. If the ARIA label really is only created when the descriptor is given, then we're fine as-is. But if it's created regardless, then we should delete "when a descriptor is given." (Or change it to something like ", including descriptor" or make it a second seetence like "...for a filter button. Includes a descriptor when provided.")

---

So I think that what you are saying is good feedback. However, I don't know that the change you are suggesting should go here. These descriptions are used for translators and don't 1:1 coorelate to the end product. For example, we do use `Filter` when no descriptor is given but we use the `skyux_filter_button_title` string of `Filter` when one is not given as the `aria-label`. 

I recognize that when we reuse strings in that way it can lead to this resource file getting confusing - but we also don't want to retranslate something when it isn't needed due to cost. 

So in this case - I think the description is ok because we are describing the string and not the `aria-label` behavior and the string is only used when the descriptor is given.

---

Makes sense. Thanks for the explanation!